### PR TITLE
[GH-2] Support for RPM based installation

### PR DESCRIPTION
**Pull request by daharon**
### Description

Support for Red Hat based distributions.
### Issues Resolved

GH-2
### Check List
- [X] All tests pass. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD
- [x] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [x] The CLA has been signed. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD

### CODE REVIEW

**Comment by daharon** [comment]
@tas50 Can you please take a look?

**Comment by ernestas-poskus** [comment]
@tas50 @mivok @tmonk42 

what is the status of this

File: recipes/install.rb [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    uri 'https://packagecloud.io/grafana/stable/debian/'
+    distribution 'wheezy' # This is used for all debian/ubuntu versions
+    key 'https://packagecloud.io/gpg.key'
+    components ['main']
+    action :add
+  end
+when 'rhel'
+  yum_repository 'grafana' do
+    description 'Grafana - Stable'
+    baseurl 'https://packagecloud.io/grafana/stable/el/$releasever/$basearch'
```
Comment (iennae) [inline recipes/install.rb]: $releasever requires centos-release package. In the yum cookbooks we've been updating to use #{node['platform_version'].to_i} instead.

**Review by iennae — CHANGES_REQUESTED** [review]
Can you rebase to master and update the use of $releasever? Thanks so much for this contribution.

**Review by iennae — APPROVED** [review]

**Comment by iennae** [comment]
Thanks so much for the contribution!

**Comment by fletchowns** [comment]
Thank you @daharon !

**Pull request merged**